src/test/java/de/tum/in/www1/artemis/ArchitectureTest.java

```diff
@@ -201,4 +207,28 @@ public void check(JavaMethod item, ConditionEvents events) {
             }
         };
     }
+
+    private ArchCondition<JavaMethod> useUpperCaseSQLStyle() {
+        return new ArchCondition<>("@Query content should follow the style guide") {
+
+            @Override
+            public void check(JavaMethod item, ConditionEvents events) {
+                var queryAnnotation = item.getAnnotations().stream().filter(simpleNameAnnotation("Query")).findAny();
+                if (queryAnnotation.isEmpty()) {
+                    return;
+                }
+                Object valueProperty = queryAnnotation.get().getExplicitlyDeclaredProperty("value");
+                if (!(valueProperty instanceof String query)) {
+                    return;
+                }
+                var checkedKeyword = Set.of("SELECT", "DISTINCT", "EXISTS", "FROM", "WHERE", "JOIN", "FETCH", "TREAT", "AND", "ORDER BY", "GROUP BY", "COUNT", "FALSE", "TRUE",
+                        "NULL", "LIKE");
+                for (var keyword : checkedKeyword) {
+                    if (StringUtils.containsIgnoreCase(query, keyword) && !query.contains(keyword)) {
```

Comment:
This does not work if the query string inconsistently uses the keywords. Example:
```sql
SELECT * FROM xyz1
JOIN xyz2
join xyz3
```
This passes the check, despite containing a lower case join

It could also cause issues if one of these is used as part of something else. Example
```sql
SELECT * FROM random
```
This would fail the test, as it contains `and` in lower case